### ISSUE TITLE
`distloss` adjustement is applied twice to `distpv` profiles

### ISSUE THREAD

**Issue by wesleyjcole** [ReEDS-Model/ReEDS#202]
### Summary
Because distributed pv (`distpv`) generation is at the point of consumption (end-use load), but ReEDS uses busbar load, we increase `distpv` profiles by the distribution loss rate (`distloss`).  That adjustment is applied twice in the model.  Once in `recf.py`:

```
    ### Scale up distpv by 1/(1-distloss)
    recf.loc[
        :, resources.loc[resources.tech.isin(vre_dist),'resource'].values
    ] /= (1 - distloss)
```

and again in `2_temporal_params.gms`:

```
* distpv capacity factor is divided by (1.0 - distloss) to provide a busbar equivalent capacity factor
m_cf(i,v,r,h,t)$[distpv(i)$valcap(i,v,r,t)] = m_cf(i,v,r,h,t) / (1.0 - distloss) ;
```

The double-counting is confirmed via `distpv` capacity factor values for p1:

`inputs/profiles_cf/cf_distpv_county.h5`: Max CF = 0.826 (ILR-adjusted maximum, 1/1.21 = 0.826)
`inputs/profiles_cf/cf_distpv_county.h5`: Max CF after aggregating counties to p1 = 0.7824
`inputs_case/recf.h5`: Max CF = 0.8237 ( = 0.7824 / (1 - 0.05), small float difference because of float16)
`m_cf` in GAMS: Max CF = .8671 ( = 0.8237 / (1 - 0.05))

There are a few lingering `cf_gather.py` references related to inputs here, so I think this double-application just appeared somewhere in our transition from `cf_gather.py` to our new configuration.

There are two ways to fix this:

1. Remove the `distloss` adjustment in `2_temporal_params.gms`
2. decrease the CF by `distloss` in `hourly_writetimeseries.py`

Option 1 is more straightforward, and then there wouldn't be three adjustments.  Option 2 would mean that the profiles in `inputs_case/rep` would be consistent with `load_allyear.csv` (which is in end-use terms), which could create inconsistencies or problems in other venues. (Removing the `distloss` adjustment in `recf.py` is not an option because that would create inconsistencies in PRAS)

There might be a third option where we rethink the entire `distloss` application and make it more straightforward throughout, but I haven't thought through that one yet.

I also think we could likely create a test that would check for things like this, so that it doesn't unintentionally slip through.

**Comment by wesleyjcole** [comment]
Actually, after asking Claude, I think a refactor option would be best, and would reduce inconsistencies and confusion:

site | now | after
-- | -- | --
hourly_load.py:507 | /(1-dl) → busbar | keep — the one canonical load conversion
recf.py:470 | distpv /(1-dl) → busbar | keep — the one canonical distpv conversion
hourly_writetimeseries.py:208 | *(1-dl) down | delete
2_temporal_params.gms:765 | /(1-dl) up | delete → load_exog = load_allyear
2_temporal_params.gms:837, 852 | /(1-dl) up | delete
2_temporal_params.gms:520 | distpv /(1-dl) up | delete
compare_cases.py:633-639 | hardcoded distloss = 0.05, grosses peak_ccseason | delete — file is already busbar, and the hardcoded scalar goes with it
c_model / report.gms end-use conversions | busbar→end-use at point of use | unchanged

`load_allyear` and `peak_ccseason` are labeled "--MW-- end-use" and would become busbar. `peak_h` is already declared "--MW-- busbar peak demand" while today it holds end-use values, so the refactor would make the existing declaration true.

`uncertainty_plots.py` would also plot busbar load instead of end-use load.  If that is intended to show end-use load, then a `distloss` adjustment would be needed there.

A refactor could also lead to different timeslice selections, as the load would be 5% different going into the representative period clustering/optimization.
